samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -565,9 +662,11 @@ static void broadcast_config_clear(void)
 {
 	int ret;
 
-	ret = lc3_streamer_close_all_streams();
-	if (ret) {
-		LOG_ERR("Failed to close all LC3 streams: %d", ret);
+	if (!sd_card_not_present) {
```

Comment:
Double negative. It might be better to rename the bool to sd_card_present and then flip the logic